lookout/style/typos/cmdline_tools.py

```diff
@@ -175,6 +176,20 @@ def add_parser(name, help):
               recommended to set this parameter if there are more than 20 repositories \
               in the dataset. Temporary directory is used if not set.")
 
+    # Report for typos on identifiers datasets
+    typos_report_parser = add_parser("typos-identifiers-report",
```

Comment:
```suggestion
    typos_report_parser = add_parser("evaluate-fixes",
```

"evaluate-noise" for testing on correct identifiers